grafana-plugin/src/models/filters/filters.helpers.ts

```diff
@@ -2,7 +2,6 @@ export const getApiPathByPage = (page: string) => {
   return (
     {
       outgoing_webhooks: 'custom_buttons',
-      outgoing_webhooks_2: 'webhooks',
```

Comment:
Turns out we don't pass `outgoing_webhooks_2` into this and rely on the fallback, webhooks is being passed in and returned.